docs/AboutUs.adoc

```diff
@@ -28,8 +28,8 @@ Responsibilities: Model
 '''
 
 === Ong Han Sheng
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+image::99448783.png[width="150", align="left"]
```

Comment:
can you change the name of your picture to your github id? will be much clearer.

Reply:
Okay sure thanks